docs/source/inference.mdx

```diff
@@ -96,13 +96,13 @@ tokenizer.save_pretrained(save_directory)
 
 ### Weight only quantization
 
-You can also apply INT8 quantization on your models weights when exporting your model with the CLI:
+You can also apply 8-bit or 4-bit weight quantization when exporting your model with the CLI:
 
 ```bash
-optimum-cli export openvino --model gpt2 --int8 ov_model
+optimum-cli export openvino --model gpt2 --weight-format int8 ov_model
 ```
 
-This will results in the exported model linear and embedding layers to be quantized to INT8, the activations will be kept in floating point precision.
+This will results in the exported model linear and embedding layers to be quantized to INT8, the activations will be kept in floating point precision. 
```

Comment:
```suggestion
This will results in the exported model linear and embedding layers to be quantized to INT8 or INT4, the activations will be kept in floating point precision.
```